src/anm/analytics.js

```diff
@@ -0,0 +1,76 @@
+var engine = require('engine');
+
+var Analytics = function () {
+    var self = this,
+        supportSendBeacon = !!navigator.sendBeacon,
+        timeout = supportSendBeacon ? 2000 : 1000,
```

Comment:
Actually, the more I see information about it, the more it also looks very new and unestablished (not as new as idleCallback, but still :) ).